base/uk.ac.stfc.isis.ibex.ui.help/src/uk/ac/stfc/isis/ibex/ui/help/ClientAboutDialogBox.java

```diff
@@ -1,28 +1,51 @@
 package uk.ac.stfc.isis.ibex.ui.help;
 
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+
 import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.layout.GridData;
 import org.eclipse.swt.layout.GridLayout;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Label;
 import org.eclipse.swt.widgets.Shell;
+import org.eclipse.swt.graphics.GC;
 
 import uk.ac.stfc.isis.ibex.ui.about.AboutDialogBox;
 
 public class ClientAboutDialogBox extends AboutDialogBox {
+	Image image;
 
 	public ClientAboutDialogBox(Shell parentShell) {
 		super(parentShell, "IBEX");
 	}
-	
+
 	@Override
 	protected Control createDialogArea(Composite parent) {
+		try {
+			image = new Image(null, new FileInputStream(
+					"C:\\Instrument\\Dev\\ibex_gui\\base\\uk.ac.stfc.isis.ibex.ui.help\\icon\\splash200x132.bmp"));
```

Comment:
Why do we use a .bmp here and a .gif elsewhere?

Reply:
.gif version logos only has the head of the animal whilst .bmp version is the one we see in splash screen. I had to edit that .bmp logo so the sizing is more appropriate for the About window

Reply:
Right, I thought I remembered you mentioning something along those lines, but wanted to check.